docs/tutorial/installation.rst

```diff
@@ -67,3 +67,17 @@ Comment out the ``callback`` and ``bindings`` options, and all the
 
 In the ``[client_properties]`` section, change the ``app`` value to ``Fedora
 Messaging tutorial``.
+
+
+Note: It is essential to configure the `passive_declares` option correctly in the `etc/fedora-messaging/config.toml` file. This setting is mandatory for users of Fedora's `/pubsub` vhost and should be set to `true`. It controls the declaration of queues and exchanges.
+
+Ensure the following steps are taken to configure `passive_declares`:
+
+1. **Access Configuration File**: Locate the `config.toml` file for Fedora Messaging. This file is typically found at `config.toml.example`.
```

Comment:
That part is not correct, applications will usually have their Fedora Messaging config file at `/etc/fedora-messaging/config.toml". The `.example` file is just what is provided by the library as an example, that's not what client applications will actually use.

Reply:
Is it fine now ?